backend/server/src/sync/mod.rs

```diff
@@ -92,7 +92,8 @@ async fn harvest(config: &Config) -> Result<()> {
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
 struct HarvestResponse {
-    includes_items_until: Timestamp,
+    #[serde(with = "chrono::serde::ts_milliseconds")]
```

Comment:
Is there no way to factor out this definition? :thinking:

Reply:
Sure, we could create a new type that deserializes as milli seconds, but thats way more code than two of these `serde(with)` attributes. So I'm against that.